nss-day-cohort-22/Class-Website
# Style Option :: Added Tooltips that show when hovering over technology icons

## Link to Ticket
no direct ticket

## Description of Proposed Changes

- I added tool tips for each technology icon that shows the name of the technology when you hover over the icon image
- I added the name of each technology to an `<alt="">` tag for each icon image
- I removed the name of each technology from underneath the icon image. 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-AddingTooltipsToTechnologyIcons
```

Load the website via Http-server and scroll down to the list of technologies. Hover over each icon image and confirm that the name of that tech appears in a tool tip popup.


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Main.js

## Mentions @username

Tag users that need to review this code
